apps/mobile/src/screens/onboarding/libs/components/content/content.tsx

```diff
@@ -0,0 +1,51 @@
+import {
+	type Control,
+	type FieldErrors,
+	type FieldValues,
+} from "react-hook-form";
+import { type OnboardingQuestionResponseDto } from "shared";
+
+import { RadioGroup, Text } from "~/libs/components/components";
+
+import {
+	type OnboardingFormValues,
+	type RadioGroupValue,
+} from "../../types/types";
+import { styles } from "./styles";
+
+type Properties<T extends FieldValues> = {
+	control: Control<T>;
+	errors: FieldErrors<T>;
+	question: null | OnboardingQuestionResponseDto;
+};
+
+const Content: React.FC<Properties<OnboardingFormValues>> = ({
+	control,
+	errors,
+	question,
+}) => {
+	let mapData: RadioGroupValue[] = [];
+
+	if (question) {
+		mapData = question.answers.map((answer) => ({
```

Comment:
Can we make simply like this and if no mapData return `null`?
```suggestion
		const mapData = question?.answers.map((answer) => ({
```